README.md

```diff
@@ -146,3 +146,16 @@ Tests can be run using [npm][npm]:
 [witten-markets]:http://www.stadtmarketing-witten.de/einkaufen/wochenmaerkte.html
 [wuppertal-wikipedia]:https://en.wikipedia.org/wiki/Wuppertal
 [wuppertal-markets]:https://www.wuppertal.de/tourismus-freizeit/einkaufen/102370100000204430.php
+
+
+## Deployment
+When new code is pushed to master travis runs a test suite against it. If those test
+pass [fabric](fabfile.org) is used to deploy the new version to the server running
+under https://wo-ist-markt.de.
+
+The command that travis executes is `fab deploy -i markt_deploy_id_rsa -H deploy@kiesinger.okfn.de:2207`
+where markt_deploy_id_rsa is an ssh key to log into the server. It is commited to this repository in
```

Comment:
After the latest problems I would leave the paragraph in there. Please format the `where markt_deploy_id_rsa` with backticks. Please replace the last sentence with: The command to deploy manually is: `fab deploy -H deploy@kiesinger.okfn.de:2207`.